src/utils/openmrs-resource.ts

```diff
@@ -0,0 +1,5 @@
+export interface OpenmrsResource {
+  uuid: string;
+  display?: string;
+  [anythingElse: string]: any;
```

Comment:
This pretty much overrides it all. Sure you want to define it like that?

Why is this placed in `src/utils`? Wouldn't `src/types` be a better location?